### PR TITLE
Update loading state for population projection view

### PR DESCRIPTION
**Pull request by Catacola**
## Description of the change

Add a spinner for the chart and make sure there is empty space where the "last updated" message will appear.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes [#XXXX]

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/PageProjections/index.tsx [hunk 1]
```diff
@@ -49,7 +49,9 @@ const PageProjections: React.FC = () => {
           />
         }
       >
-        <PopulationSummaryMetrics isLoading={isLoading} isError={isError} />
+        <PopulationSummaryMetrics isLoading isError={isError} />
```
Comment (jovergaag) [inline src/core/PageProjections/index.tsx:52]: The loading state of the `SummaryMetrics` doesn't seem to be working correctly. When I set isLoading to true for everything, the summary Metrics placeholders and titles don't seem to be correct.

File: src/core/PopulationTimeSeriesChart/index.tsx [hunk 2]
```diff
@@ -39,13 +40,25 @@ type PlotLine = {
...
+const PopulationTimeSeriesChart: React.FC<Props> = ({ isLoading = false }) => {
   const view = getViewFromPathname(useLocation().pathname);
   const compartment: SimulationCompartment = getCompartmentFromView(view);
   const { metricsStore, filtersStore } = useCoreStore();
   const { gender, legalStatus } = filtersStore.filters;
   const filteredData = metricsStore.projections.getFilteredDataByView(view);
 
+  if (isLoading) {
+    return (
+      <div className="PopulationTimeSeriesChart PopulationTimeSeriesChart--Loading">
```
Comment (jovergaag) [inline src/core/PopulationTimeSeriesChart/index.tsx:56]: The modifier in BEM is usually lowercase. `PopulationTimeSeriesChart--loading`

**Review by jovergaag — CHANGES_REQUESTED** [review]

**Comment by jovergaag** [comment]
Also - I was getting errors when trying to run this locally, due to the `simulationDate`. Both on staging data and in demo mode.

**Comment by Catacola** [comment]
I see what happened. I updated the diff below this one which caused errors in this one. All should be fixed now.

**Review by jovergaag — APPROVED** [review]

**Pull request merged**